mustache.js

```diff
@@ -578,25 +589,25 @@
     return buffer;
   };
 
-  Writer.prototype.renderSection = function renderSection (token, context, partials, originalTemplate) {
+  Writer.prototype.renderSection = function renderSection (token, context, partials, originalTemplate, config) {
     var self = this;
     var buffer = '';
     var value = context.lookup(token[1]);
 
     // This function is used to render an arbitrary template
     // in the current context by higher-order sections.
     function subRender (template) {
-      return self.render(template, context, partials);
+      return self.render(template, context, partials, config);
```

Comment:
Bugfix: `tags` was not being passed through before. Now a `tags` array or a `config` object will be passed via the `render` function argument in a lambda section using the `function (text, render)` API.